foundations/html_css/html-foundations/html-boilerplate.md

```diff
@@ -78,70 +77,70 @@ There are many more elements that can go within the head of an HTML document. Ho
 
 Back in our `index.html` file, let's add a head element with a charset meta element and a title within it. The head element goes within the HTML element and should always be the first element under the opening `<html>` tag:
 
-~~~html
+```html
 <!DOCTYPE html>
 
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <title>My First Webpage</title>
-  </head>
+    <head>
+        <meta charset="UTF-8" />
+        <title>My First Webpage</title>
+    </head>
```

Comment:
Is there a particular reason for adding an additional 2 whitespace characters here?

Reply:
I did not realize I did, my mistake. I did look over the other files, to make sure they adhered to the style guide, and I must have mis clicked at some point. Sorry:(